packages/document-model-libs/editors/chat-room-editor/components/Message/Message.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react/jsx-no-bind */
```

Comment:
these are supposed to just be warnings, but if they are annoying you then let's just disable them in the eslint config. they are just stylistic anyway.